runscripts/awicm3/v3.2/awicm3-v3.2-levante-TCO95L91-CORE2.yaml

```diff
@@ -5,10 +5,10 @@ general:
     account: "ab0246"
     compute_time: "00:20:00"
     initial_date: "2000-01-01"
-    final_date: "2000-02-01"
+    final_date: "2000-01-02"
```

Comment:
Hey @JanStreffing, are you sure you want all this changes in your runscript? Daily instead of monthly?

Reply:
Hi @JanStreffing , @mandresm ,
are you still doing changes on this PR or is it still in review? Or can we merge it?